<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <title>Lingbin Ni | Genome Assembly of Soybean Accession</title>
  <meta name="description" content="Personal website of Lingbin Ni.">

  <!-- Fonts and Icons -->
  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons" />

  <!-- CSS Files -->
  <link rel="stylesheet" href="/assets/css/all.min.css">
  <link rel="stylesheet" href="/assets/css/academicons.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">
  <link rel="canonical" href="/projects/research_ZH13/">
</head>
<body>
  <!-- Header -->
  <nav id="navbar" class="navbar fixed-top navbar-expand-md grey lighten-5 z-depth-1 navbar-light">
    <div class="container-fluid p-0">

        <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Lingbin</span> Ni</a>

      <button class="navbar-toggler ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <li class="nav-item ">
            <a class="nav-link" href="/">
              About

            </a>
          </li>




              <li class="nav-item ">
                  <a class="nav-link" href="/cv/">
                    Curriculum Vitae

                  </a>
              </li>





              <li class="nav-item ">
                  <a class="nav-link" href="/projects/">
                    Projects

                  </a>
              </li>



              <li class="nav-item ">
                  <a class="nav-link" href="/publications/">
                    Publications

                  </a>
              </li>



              <li class="nav-item ">
                  <a class="nav-link" href="/activities/">
                    Activities

                  </a>
              </li>




        </ul>
      </div>
    </div>
  </nav>

  <!-- Scrolling Progress Bar -->
  <progress id="progress" value="0">
    <div class="progress-container">
      <span class="progress-bar"></span>
    </div>
  </progress>

  <!-- Content -->
  <div class="content">
    <nav aria-label="breadcrumb">
  <ol class="breadcrumb p-0 text">
    <li class="breadcrumb-item"><a href="/">Home</a></li>
    <li class="breadcrumb-item"><a href="/projects">Projects</a></li>
    <li class="breadcrumb-item active">Genome Assembly of a Chinese Soybean</li>
  </ol>
</nav>

<div class="row m-0" style="width: 100%;">
  <div class="col-sm-12 p-0">
    <h1>Genome Assembly of a Chinese Soybean</h1>
    <h5 class="mb-4">De novo assembly and sequence analysis of a Chinese soybean genome</h5>
  </div>
  <!-- <div class="col-sm-2 p-0 mt-auto mb-auto">
    <img src="/assets/img/tensorflow_for_scala_logo.svg" alt="project thumbnail">
  </div> -->
</div>

<div class="container-fluid p-0 text-justify">
  <p>
  Soybean is one of the most important crops worldwide. A high-quality reference genome will facilitate its functional analysis and molecular breeding. Previously, we de novo assembled a high-quality Chinese soybean genome Gmax_ZH13. However, due to technical limitations at the time when we generated Gmax_ZH13, a large number of small contigs were not anchored onto chromosomes. Therefore, we here build a new golden reference genome for Zhonghuang 13 consisting of 20 nearly complete chromosomes by adding more single-molecule real time (SMRT) sequencing reads. Furthermore, we add large RNA-seq and smRNA-seq datasets for improving the annotation of its protein coding genes.
  </p>

  <p>
  In this project, I mainly conducted the tissue collection, library preparation, and further analysis for RNA-seq and smRNA-seq.
  </p>

<h2 id="main-features">Main Workflow</h2>

<ul>
  <li>
    <p>Tissue collection:</p>
  </li>

  <p>
  To improve the accuracy of gene annotation, 27 tissues were collected from Zhonghuang 13 plants during their growing season at the experimental station of the Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, in 2018. Each tissues have two replications.
  </p>

  <li>
    <p>Library preparation:</p>
  </li>

  <p>
  Total RNA of each sample was isolated using RNA isolation kit (Tiangen, Beijing, China) according to the manufacturerâ€™s protocol. Libraries for RNA-seq were constructed and sequenced on the Illumina HiSeq X Ten platform for each sample. Libraries for smRNA-seq were constructed and sequenced on the Illumina NextSeq 550AR platform. In total, 353 Gb reads were produced.
  </p>

  <li>
    <p>Protein coding genes annotation:</p>
  </li>

  <p>
  The RNA-seq data, together with the Iso-seq reads used for annotation in the last version, were used as expression sequence tag (EST) evidences to predict protein coding genes using MAKER, which was a professional pipeline popularly used in gene prediction. We annotated a total of 55,443 protein coding genes containing 96,366 mRNAs in the nuclear genome. We found that 97% of the 1,440 single copy Embryophyta genes in BUSCO_v3 were completely assembled, confirming the high quality of our annotation. We found that 42,259 of the newly annotated genes matching to the genes in the last version, whose IDs were therefore inherited from Gmax_ZH13. In addition, we annotated 81 and 49 protein coding genes for chloroplast genome and mitochondrial genome respectively.
  </p>

  <li>
    <p>MIRNA genes annotation:</p>
  </li>

  <p>
  To annotate MIRNA genes, small RNAs from the same 27 samples used for protein coding gene annotation were sequenced. Using these reads, we annotated 331 MIRNA genes. Transcripts of these loci generated 349 pair of miRNAs (miRNA-3p and miRNA-5p). Using the RNA sequence and small RNA sequence data of 27 samples, we provided a detailed expression profiling for all the protein coding genes and miRNAs.
  </p>

</ul>

<h2 id="References">References</h2>

  <div class="col mt-4 p-0" style="color: #78909c; font-size: 14px;">
  Shen Y, Du H, Liu Y, Ni L, Wang Z, Liang C, Tian Z. (2019). <a href="https://link.springer.com/article/10.1007/s11427-019-9822-2" target="_blank"><i>Update soybean Zhonghuang 13 genome to a golden reference</i></a>. Sci China Life Sci 62, 1257-1260.
  </div>

<h2 id="funding">Funding</h2>

<p>Funding for the de novo assembly and sequence analysis of the genome has been generously provided by the following sponsors:</p>

<div class="col table-responsive">
<table class="table table-sm table-borderless">
  <tr>
    <td class="p-1 text-center align-middle" style="width: 30%;">
      <img class="img-responsive" style="width: 80%;" src="/assets/img/CAS.jpg" alt="cmu_logo" />
    </td>
    <td class="p-1 text-center align-middle" style="width: 30%;">
      <img class="img-responsive" style="width: 80%;" src="/assets/img/NSFC.jpeg" alt="cmu_logo" />
    </td>
    <td class="p-1 text-center align-middle" style="width: 30%;">
      <img class="img-responsive" style="width: 80%;" src="/assets/img/PCCE.png" alt="cmu_logo" />
    </td>
  </tr>
  <tr>
    <td class="p-1 font-weight-light text-center"><b>CMU Presidential Fellowship</b> <br />awarded to <br />Emmanouil Antonios Platanios</td>
    <td class="p-1 font-weight-light text-center"><b>National Science Foundation</b> <br />Grant #: IIS1250956</td>
    <td class="p-1 font-weight-light text-center"><b>Air Force Office of Scientific Research</b> <br />Grant #: FA95501710218</td>
  </tr>
</table>
</div>

</div>

  </div>

  <!-- Footer -->
  <footer>
    &copy; Copyright 2023 Lingbin Ni
  </footer>

  <!-- Core JavaScript Files -->
  <script src="/assets/js/jquery.min.js" type="text/javascript"></script>
  <script src="/assets/js/popper.min.js" type="text/javascript"></script>
  <script src="/assets/js/bootstrap.min.js" type="text/javascript"></script>
  <script src="/assets/js/mdb.min.js" type="text/javascript"></script>
  <script async="" src="https://cdnjs.cloudflare.com/ajax/libs/masonry/4.2.2/masonry.pkgd.min.js" integrity="sha384-GNFwBvfVxBkLMJpYMOABq3c+d3KnQxudP/mGPkzpZSTYykLBNsZEnG2D9G/X/+7D" crossorigin="anonymous"></script>
  <script src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML"></script>
  <script src="/assets/js/common.js"></script>

  <!-- GitHub Stars -->
  <script src="/assets/js/github-stars.js"></script>
  <script type="text/javascript">









        githubStars("eaplatanios/symphony-mt", function(stars) { $("#curriculum-learningeaplatanios-symphony-mt-stars").text('' + stars); });




        githubStars("eaplatanios/jelly-bean-world", function(stars) { $("#jelly-bean-worldeaplatanios-jelly-bean-world-stars").text('' + stars); });




        githubStars("eaplatanios/symphony-mt", function(stars) { $("#machine-translationeaplatanios-symphony-mt-stars").text('' + stars); });








        githubStars("eaplatanios/tensorflow_scala", function(stars) { $("#TensorFlow-Scalaeaplatanios-tensorflow_scala-stars").text('' + stars); });


  </script>

  <!-- Scrolling Progress Bar -->
  <script type="text/javascript">
    $(document).ready(function() {
      var navbarHeight = $('#navbar').outerHeight(true);
      $('body').css({ 'padding-top': navbarHeight });
      $('progress-container').css({ 'padding-top': navbarHeight });
      var progressBar = $('#progress');
      progressBar.css({ 'top': navbarHeight });
      var getMax = function() { return $(document).height() - $(window).height(); }
      var getValue = function() { return $(window).scrollTop(); }
      // Check if the browser supports the progress element.
      if ('max' in document.createElement('progress')) {
        // Set the 'max' attribute for the first time.
        progressBar.attr({ max: getMax() });
        progressBar.attr({ value: getValue() });

        $(document).on('scroll', function() {
          // On scroll only the 'value' attribute needs to be calculated.
          progressBar.attr({ value: getValue() });
        });

        $(window).resize(function() {
          var navbarHeight = $('#navbar').outerHeight(true);
          $('body').css({ 'padding-top': navbarHeight });
          $('progress-container').css({ 'padding-top': navbarHeight });
          progressBar.css({ 'top': navbarHeight });
          // On resize, both the 'max' and 'value' attributes need to be calculated.
          progressBar.attr({ max: getMax(), value: getValue() });
        });
      } else {
        var max = getMax(), value, width;
        var getWidth = function() {
          // Calculate the window width as a percentage.
          value = getValue();
          width = (value/max) * 100;
          width = width + '%';
          return width;
        }
        var setWidth = function() { progressBar.css({ width: getWidth() }); };
        setWidth();
        $(document).on('scroll', setWidth);
        $(window).on('resize', function() {
          // Need to reset the 'max' attribute.
          max = getMax();
          setWidth();
        });
      }
    });
  </script>

  <!-- Code Syntax Highlighting -->
  <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro" rel="stylesheet">
  <script src="/assets/js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>

  <!-- Script Used for Randomizing the Projects Order -->
  <!-- <script type="text/javascript">
    $.fn.shuffleChildren = function() {
      $.each(this.get(), function(index, el) {
        var $el = $(el);
        var $find = $el.children();

        $find.sort(function() {
          return 0.5 - Math.random();
        });

        $el.empty();
        $find.appendTo($el);
      });
    };
    $("#projects").shuffleChildren();
  </script> -->

  <!-- Project Cards Layout -->
  <script type="text/javascript">
    var $grid = $('#projects');

    // $grid.masonry({ percentPosition: true });
    // $grid.masonry('layout');

    // Trigger after images load.
    $grid.imagesLoaded().progress(function() {
      $grid.masonry({ percentPosition: true });
      $grid.masonry('layout');
    });
  </script>

  <!-- Enable Tooltips -->
  <script type="text/javascript">
    $(function () {
      $('[data-toggle="tooltip"]').tooltip()
    })
  </script>

  <!-- Google Analytics -->
  <script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
    ga('create', 'UA-54519238-1', 'auto');
    ga('send', 'pageview');
  </script>
</body>
</html>
