<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <title>Lingbin Ni | Genome Assembly of Soybean Accession</title>
  <meta name="description" content="Personal website of Lingbin Ni.">

  <!-- Fonts and Icons -->
  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons" />

  <!-- CSS Files -->
  <link rel="stylesheet" href="/assets/css/all.min.css">
  <link rel="stylesheet" href="/assets/css/academicons.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">
  <link rel="canonical" href="/projects/research_3Dgenome/">
</head>
<body>
  <!-- Header -->
  <nav id="navbar" class="navbar fixed-top navbar-expand-md grey lighten-5 z-depth-1 navbar-light">
    <div class="container-fluid p-0">

        <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Lingbin</span> Ni</a>

      <button class="navbar-toggler ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <li class="nav-item ">
            <a class="nav-link" href="/">
              About

            </a>
          </li>




              <li class="nav-item ">
                  <a class="nav-link" href="/cv/">
                    Curriculum Vitae

                  </a>
              </li>





              <li class="nav-item ">
                  <a class="nav-link" href="/projects/">
                    Projects

                  </a>
              </li>



              <li class="nav-item ">
                  <a class="nav-link" href="/publications/">
                    Publications

                  </a>
              </li>



              <li class="nav-item ">
                  <a class="nav-link" href="/teaching/">
                    Activities

                  </a>
              </li>




        </ul>
      </div>
    </div>
  </nav>

  <!-- Scrolling Progress Bar -->
  <progress id="progress" value="0">
    <div class="progress-container">
      <span class="progress-bar"></span>
    </div>
  </progress>

  <!-- Content -->
  <div class="content">
    <nav aria-label="breadcrumb">
  <ol class="breadcrumb p-0 text">
    <li class="breadcrumb-item"><a href="/">Home</a></li>
    <li class="breadcrumb-item"><a href="/projects">Projects</a></li>
    <li class="breadcrumb-item active">3D Genomic Diversity in Plant Accessions</li>
  </ol>
</nav>

<div class="row m-0" style="width: 100%;">
  <div class="col-sm-12 p-0">
    <h1>3D Genomic Diversity in Plant Accessions</h1>
    <h5 class="mb-4">The first pan-3D genome related to domestication and improvement in plants</h5>
  </div>
  <!-- <div class="col-sm-2 p-0 mt-auto mb-auto">
    <img src="/assets/img/projects/machine_translation/machine_translation.svg" alt="project thumbnail">
  </div> -->
</div>

<div class="container-fluid p-0 text-justify">
  <p>During my PhD I worked on a couple of interesting projects related to 3D genomic diversity in plants. In this page I plan to talk about two of them: one which introduced the idea of pan-3D genome in plant accessions and one which performs a detailed investigation between 3D genomic diversity and single nucleotide polymorphisms (SNPs). The following post is currently about the first project, but it will soon be updated with information about the second.</p>

<div class="alert alert-danger" role="alert">
  <b>NOTE:</b> The following article only contains part of the results in <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02854-8/" target="_blank">the pan-3D genome paper</a>.
</div>

<div class="container-fluid p-0">
  <img class="img-responsive col-12" src="/assets/img/overview_3Dgenome.png" alt="overview figure" />
  <h6 class="font-italic text-center" style="color: #78909c;"><u>Figure 1:</u> Overview of the contextual parameter generator that is introduced in this post. The top part of the figure shows a typical neural machine translation system (consisting of an encoder and a decoder network). The bottom part, shown in red, shows our parameter generator component.</h6>
</div>

<div class="col mt-4 p-0">
  3D genome organization can bring distal cis-regulatory elements (CREs) into spatial proximity with their target genes and plays an important role in gene regulation. High-throughput studies have shed light on a hierarchical organization of the 3D genome, which can be arranged into chromosome territories, compartments, topologically associating domains (TADs), and chromatin loops. Recent analyses have suggested that 3D genomes may serve as an important driving force during cell differentiation, embryo development, and genome evolution. However, the diversity of 3D genomes across plant accessions are seldom reported.
</div>

<div class="container-fluid text-center mt-4 p-0">
  <img class="img-responsive col-12 col-sm-10 col-md-6 ml-auto mr-auto" src="/assets/img/3Dgenome-pipeline.png" alt="overview figure" />
  <h6 class="font-italic text-center mt-2" style="color: #78909c;"><u>Figure 2:</u> Illustration of the machine translation problem.</h6>
</div>

<div class="col mt-4 p-0">
  In this study, we perform the pan-3D genome analysis using Hi-C sequencing data from 27 soybean accessions and comprehensively investigate the relationships between 3D genomic variations and structural variations (SVs) as well as gene expression. This project represents the first studies of 3D genomic diversity investigations across multiple accessions in plant species, and provide a deeper insight into genome evolution and gene regulation in plant accessions.
</div>

<h3 class="title mt-4 p-0 text-left">3D genome analysis</h3>

<div class="col mt-4 p-0">
  To investigate the 3D genome organization in soybean, we performed in situ Hi-C sequencing of the 27 accessions which included 3 wild soybeans, 9 landraces, and 15 improved cultivars. The in situ Hi-C experiment for each accession was designed with two biological replicates, and each replicate produced an average of ~470 million raw read pairs and ~100 million unique long-range (more than 20 kb) cis contacts. The two biological replicates of each accession were highly reproducible. Complementing these Hi-C data, we profiled the transcriptome through RNA sequencing (RNA-seq) in these accessions.
</div>

<div class="container-fluid text-center mt-4 p-0">
  <div class="row m-0">
    <div class="col-12 col-sm-4 mb-3 mb-sm-0">
      <img class="img-responsive" style="width: 100%;" src="/assets/img/projects/machine_translation/2_pairwise.svg" alt="overview figure" />
    </div>
    <div class="col-12 col-sm-4 mb-3 mb-sm-0">
      <img class="img-responsive" style="width: 100%;" src="/assets/img/projects/machine_translation/3_universal.svg" alt="overview figure" />
    </div>
    <div class="col-12 col-sm-4 mb-1 mb-sm-0">
      <img class="img-responsive" style="width: 100%;" src="/assets/img/projects/machine_translation/4_per_language.svg" alt="overview figure" />
    </div>
  </div>
  <h6 class="font-italic text-center mt-2" style="color: #78909c;"><u>Figure 3:</u> Overview of existing approaches for multilingual neural machine translation.</h6>
</div>

<div class="col mt-4 p-0">
  Assuming that we have \(L\) languages and \(P\) trainable parameters per NMT model (these could be, for example, the weights and biases of a recurrent neural network) that translates from a single source language to a single target language, we have:
</div>

<ol class="mt-2">
  <li>
    <b>Pairwise:</b> Use a pairwise NMT model per language pair. This results in \(L^2\) separate models, each with \(P\) parameters. The main issue with this approach is that no information is shared between different languages, and even between models translating from English to other languages, for example. This is especially problematic for low-resource languages, where we have very little training data available, and would ideally want to leverage the high availability of data for other languages, such as English.
  </li>
  <li>
    <b>Universal:</b> Use a single NMT model for all language pairs. In this case, an artificial word can be added to the source sentence denoting the target language for the translation. This results in a single model with \(P\) parameters, since the whole model is shared across all languages. This approach can result in overfitting to the high-resource languages. It can also limit the expressivity of the translation model for languages that are more "different" (e.g., for Turkish, if training using Italian, Spanish, Romanian, and Turkish). Ideally, we want something in-between pairwise and universal models.
  </li>
  <li>
    <b>Per-Language Encoder/Decoder:</b> Use a separate encoder and a separate decoder for each language. This results in a total of \(LP\) parameters and lies somewhere between the pairwise and the universal approaches. However, no information can be shared between languages that are similar (e.g., Italian and Spanish).
  </li>
</ol>

<div class="col mt-2 p-0">
  At <a href="http://aclweb.org/anthology/D18-1039" target="_blank">EMNLP 2018</a>, we introduced the <i>contextual parameter generator (CPG)</i>, a new way to share information across different languages that generalizes the above three approaches, while mitigating their issues and allowing explicit control over the amount of sharing.
</div>

<h3 class="title mt-4 p-0 text-left">A/B compartment</h3>

<div class="col mt-4 p-0">
  Using the in situ Hi-C sequencing data, we checked the organization of A/B compartments and found certain differentiation of A/B compartments within chromosomes. For example, we observed differences around pericentromeric heterochromatin of chromosome 18 in SoyC13 and SoyC03, and variations in the euchromatin arms of chromosome 11 in SoyW02 and SoyC13. Despite these differences, the average A compartment percentages were relatively consistent across individual accessions. Collectively, these results reveal that the overall percentages of compartments are conserved, but the reorganization of A/B compartments within chromosomes occurs frequently in soybean accessions.
</div>

<div class="col mt-2 p-0">
  Nevertheless, we found that the repeat percentage of the A compartment on several chromosomes was significantly higher than the average level, such as chromosome 1 of SoyC06. Tracing these A compartments showed that the highly repetitive regions mainly came from the regions of intersection between A compartments and B compartments, which were referred to as I regions in this study. We found the A compartments and B compartments from I regions also presented intermediate status. Further investigation using the data from 27 accessions revealed that the GC contents, TSS densities, and repeat percentages of I regions (including the A compartments or B compartments from I regions) exhibited an intermediate status between the A regions and the B regions. Overall, these data illustrate that I regions exhibit intermediate genomic features.
</div>

<h4 class="title mt-4 p-0 text-left">A/B compartment</h4>

<div class="col mt-4 p-0">
  Using the in situ Hi-C sequencing data, we checked the organization of A/B compartments and found certain differentiation of A/B compartments within chromosomes. For example, we observed differences around pericentromeric heterochromatin of chromosome 18 in SoyC13 and SoyC03, and variations in the euchromatin arms of chromosome 11 in SoyW02 and SoyC13. Despite these differences, the average A compartment percentages were relatively consistent across individual accessions. Collectively, these results reveal that the overall percentages of compartments are conserved, but the reorganization of A/B compartments within chromosomes occurs frequently in soybean accessions.
</div>

<div class="col mt-2 p-0">
  Nevertheless, we found that the repeat percentage of the A compartment on several chromosomes was significantly higher than the average level, such as chromosome 1 of SoyC06. Tracing these A compartments showed that the highly repetitive regions mainly came from the regions of intersection between A compartments and B compartments, which were referred to as I regions in this study. We found the A compartments and B compartments from I regions also presented intermediate status. Further investigation using the data from 27 accessions revealed that the GC contents, TSS densities, and repeat percentages of I regions (including the A compartments or B compartments from I regions) exhibited an intermediate status between the A regions and the B regions. Overall, these data illustrate that I regions exhibit intermediate genomic features.
</div>

<h4 class="title mt-4 p-0 text-left">TAD boundary</h4>

<div class="col mt-4 p-0">
  To identify TADs in soybean, the reported “insulation score” method was adopted in this study. In parallel, contact domains and directionality indexes were also calculated. Although boundaries exhibited some variations across the 27 accessions, they had comparable normalized numbers. The identification results showed that the median size of TADs in soybean was approximately 475 kb.
</div>

<div class="col mt-2 p-0">
  We analyzed the enrichment of repeats around TAD boundaries. Meta-analysis revealed that repeats were depleted around boundaries. In contrast to humans, LTRs are the dominant TEs in plants. In soybean, non-LTR TEs accounted for less than 5% of the TEs, while LTRs accounted for ~80%. Subsequently, we investigated the enrichment of different repeat types. Interestingly, although most repeats were found to be depleted around boundaries, LINEs and SINEs were found to be enriched around boundaries particularly. These results were further confirmed through the analyses of each of the 27 soybean accessions. It has been found that most identified LINEs in plants come from the L1 and RTE superfamilies. Enrichment analysis of superfamilies showed that the length of the L1 fragment was longer around boundaries.
</div>

<h3 class="title mt-4 text-left p-0">Pan-3D genome analysis</h3>

<h3 class="title mt-4 p-0 text-left">Where should you look to learn more?</h3>

<div class="col mt-4 p-0">
  More details can be found in our pan-3D genome paper. The sequencing data generated in this study are available at the Genome Sequence Archive (GSA, https://bigd.big.ac.cn/gsa/; accession number: PRJCA009364). Computational scripts used for data analyses conducted as part of this study are available under MIT license at Github: https://github.com/LingbinNi/soybean_pan-3D_genome_analysis and Zenodo: https://doi.org/10.5281/zenodo.7514511. Custom UCSC chain files between ZH13 and 26 query genomes: https://figshare.com/articles/dataset/UCSC_chain_files_of_soybean_genomes/20027336.

  More details can be found in our <a href="http://aclweb.org/anthology/D18-1039" target="_blank">EMNLP 2018 paper</a>. We have also released an implementation of our approach and experiments as part of a new <a href="https://github.com/eaplatanios/symphony-mt" target="_blank">Scala framework for machine translation</a>. It is built on top of <a href="https://github.com/eaplatanios/tensorflow_scala" target="_blank">TensorFlow Scala</a> and follows a modular NMT design that supports various NMT models, including our baselines. It also contains data loading and preprocessing pipelines that support multiple datasets and languages, and is more efficient than other packages (e.g., <a href="https://github.com/tensorflow/nmt" target="_blank">tf-nmt</a>). Furthermore, the framework supports various vocabularies, among which we provide a new implementation for the byte-pair encoding (BPE) algorithm that is 2 to 3 orders of magnitude faster than the <a href="https://github.com/rsennrich/subword-nmt" target="_blank">released one</a>.
</div>

<div class="col mt-2 p-0">
  I also plan to release another blog post later on in 2019 with some follow-up work we have done on applying contextual parameter generation to other problems.
</div>

<h3 class="title mt-4 p-0 text-left">Acknowledgements</h3>

<div class="col mt-4 p-0">
  This work was supported by the National Key Research and Development Program of China (2021YFF1000101-3), the Strategic Priority Research Program of the Chinese Academy of Sciences (XDA24030501), and the National Natural Science Foundation of China (31788103, U22A20473).
</div>

<h3 class="title mt-4 p-0 text-left">References</h3>

<div class="col mt-4 p-0" style="color: #78909c; font-size: 14px;">
  Ni L, Liu Y, Ma X, Liu T, Yang X, Wang Z, Liang Q, Liu S, Zhang M, Wang Z, et al. (2023). <a href="http://aclweb.org/anthology/D18-1039" target="_blank"><i>Pan-3D genome analysis reveals structural and functional differentiation of soybean genomes</i></a>. Genome Biol 24, 12.
</div>

</div>

  </div>

  <!-- Footer -->
  <footer>
    &copy; Copyright 2023 Lingbin Ni


  </footer>

  <!-- Core JavaScript Files -->
  <script src="/assets/js/jquery.min.js" type="text/javascript"></script>
  <script src="/assets/js/popper.min.js" type="text/javascript"></script>
  <script src="/assets/js/bootstrap.min.js" type="text/javascript"></script>
  <script src="/assets/js/mdb.min.js" type="text/javascript"></script>
  <script async="" src="https://cdnjs.cloudflare.com/ajax/libs/masonry/4.2.2/masonry.pkgd.min.js" integrity="sha384-GNFwBvfVxBkLMJpYMOABq3c+d3KnQxudP/mGPkzpZSTYykLBNsZEnG2D9G/X/+7D" crossorigin="anonymous"></script>
  <script src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML"></script>
  <script src="/assets/js/common.js"></script>

  <!-- GitHub Stars -->
  <script src="/assets/js/github-stars.js"></script>
  <script type="text/javascript">









        githubStars("eaplatanios/symphony-mt", function(stars) { $("#curriculum-learningeaplatanios-symphony-mt-stars").text('' + stars); });




        githubStars("eaplatanios/jelly-bean-world", function(stars) { $("#jelly-bean-worldeaplatanios-jelly-bean-world-stars").text('' + stars); });




        githubStars("eaplatanios/symphony-mt", function(stars) { $("#machine-translationeaplatanios-symphony-mt-stars").text('' + stars); });








        githubStars("eaplatanios/tensorflow_scala", function(stars) { $("#TensorFlow-Scalaeaplatanios-tensorflow_scala-stars").text('' + stars); });


  </script>

  <!-- Scrolling Progress Bar -->
  <script type="text/javascript">
    $(document).ready(function() {
      var navbarHeight = $('#navbar').outerHeight(true);
      $('body').css({ 'padding-top': navbarHeight });
      $('progress-container').css({ 'padding-top': navbarHeight });
      var progressBar = $('#progress');
      progressBar.css({ 'top': navbarHeight });
      var getMax = function() { return $(document).height() - $(window).height(); }
      var getValue = function() { return $(window).scrollTop(); }
      // Check if the browser supports the progress element.
      if ('max' in document.createElement('progress')) {
        // Set the 'max' attribute for the first time.
        progressBar.attr({ max: getMax() });
        progressBar.attr({ value: getValue() });

        $(document).on('scroll', function() {
          // On scroll only the 'value' attribute needs to be calculated.
          progressBar.attr({ value: getValue() });
        });

        $(window).resize(function() {
          var navbarHeight = $('#navbar').outerHeight(true);
          $('body').css({ 'padding-top': navbarHeight });
          $('progress-container').css({ 'padding-top': navbarHeight });
          progressBar.css({ 'top': navbarHeight });
          // On resize, both the 'max' and 'value' attributes need to be calculated.
          progressBar.attr({ max: getMax(), value: getValue() });
        });
      } else {
        var max = getMax(), value, width;
        var getWidth = function() {
          // Calculate the window width as a percentage.
          value = getValue();
          width = (value/max) * 100;
          width = width + '%';
          return width;
        }
        var setWidth = function() { progressBar.css({ width: getWidth() }); };
        setWidth();
        $(document).on('scroll', setWidth);
        $(window).on('resize', function() {
          // Need to reset the 'max' attribute.
          max = getMax();
          setWidth();
        });
      }
    });
  </script>

  <!-- Code Syntax Highlighting -->
  <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro" rel="stylesheet">
  <script src="/assets/js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>

  <!-- Script Used for Randomizing the Projects Order -->
  <!-- <script type="text/javascript">
    $.fn.shuffleChildren = function() {
      $.each(this.get(), function(index, el) {
        var $el = $(el);
        var $find = $el.children();

        $find.sort(function() {
          return 0.5 - Math.random();
        });

        $el.empty();
        $find.appendTo($el);
      });
    };
    $("#projects").shuffleChildren();
  </script> -->

  <!-- Project Cards Layout -->
  <script type="text/javascript">
    var $grid = $('#projects');

    // $grid.masonry({ percentPosition: true });
    // $grid.masonry('layout');

    // Trigger after images load.
    $grid.imagesLoaded().progress(function() {
      $grid.masonry({ percentPosition: true });
      $grid.masonry('layout');
    });
  </script>

  <!-- Enable Tooltips -->
  <script type="text/javascript">
    $(function () {
      $('[data-toggle="tooltip"]').tooltip()
    })
  </script>

  <!-- Google Analytics -->
  <script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
    ga('create', 'UA-54519238-1', 'auto');
    ga('send', 'pageview');
  </script>
</body>
</html>
